compatibility_lib/compatibility_lib/test_dependency_highlighter.py

```diff
@@ -146,8 +146,9 @@ def setUp(self):
             self._store)
 
     def setup_test__get_update_priority(self):
-        priority = dependency_highlighter.Priority
-        level = dependency_highlighter.PriorityLevel
+        LOW = dependency_highlighter.PriorityLevel.LOW_PRIORITY
```

Comment:
We don't usually use upper case variable name inside functions.

Reply:
done